svm_controller.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, request
+import pickle
+import numpy as np
+
+app = Flask(__name__)
+
+model = pickle.load(open("svv.sav", 'rb'))
+
+@app.route("/digits", methods = ["GET"])
```

Comment:
maybe you could create a healthcheck endpoint like https://pypi.org/project/py-healthcheck/ ?